autogen/agentchat/contrib/agent_builder.py

```diff
@@ -139,26 +192,42 @@ def _create_agent(
 
         Args:
             agent_name: the name that identify the function of the agent (e.g., Coder, Product Manager,...)
-            model_name_or_hf_repo:
+            model_name_or_hf_repo: the name of the model or the huggingface repo.
             llm_config: specific configs for LLM (e.g., config_list, seed, temperature, ...).
             system_message: system prompt use to format an agent's behavior.
+            description: a brief description of the agent. This will improve the group chat performance.
             use_oai_assistant: use OpenAI assistant api instead of self-constructed agent.
             world_size: the max size of parallel tensors (in most of the cases, this is identical to the amount of GPUs).
 
         Returns:
             agent: a set-up agent.
         """
-        config_list = autogen.config_list_from_json(self.config_path, filter_dict={"model": [model_name_or_hf_repo]})
+        from huggingface_hub import HfApi
```

Comment:
The part of using local models is supposed to be moved to the oai layer. @olgavrou for awareness.